doc/ale-erlang.txt

```diff
@@ -3,6 +3,27 @@ ALE Erlang Integration                                     *ale-erlang-options*
 
 
 ===============================================================================
+dialyzer                                                  *ale-erlang-dialyzer*
+
+g:ale_erlang_syntaxerl_executable            *g:ale_erlang_dialyzer_executable*
+                                             *b:ale_erlang_dialyzer_executable*
+  Type: |String|
+  Default: `'dialyzer'`
+
+  This variable can be changed to specify the dialyzer executable.
+
+
+g:ale_erlang_plt_file                                   *g:ale_erlang_plt_file*
```

Comment:
Name this `ale_erlang_dialyzer_plt_file`, or move the documentation for this into a general section above. Have a look a the C or Python documentation for examples.